MIGRATION.MD

```diff
@@ -0,0 +1,234 @@
+# Migration from 1.x to 2.0
+
+The main changes introduced in SDK 2.0 compared to 1.x are:
+
+1. All the products (RUM, Trace, Logs, etc.) are now extracted to the different modules. That allows you to integrate only what is needed into your application.
+
+In comparison with the version 1.x where all the products were in the single artifact `com.datadoghq:dd-sdk-android:x.x.x`, now you need to add the following artifacts instead:
+
+* RUM: `com.datadoghq:dd-sdk-android-rum:x.x.x`
+* Logs: `com.datadoghq:dd-sdk-android-logs:x.x.x`
+* Trace: `com.datadoghq:dd-sdk-android-trace:x.x.x`
+* WebView Tracking: `com.datadoghq:dd-sdk-android-webview:x.x.x`
+
+**NOTE**: In case of NDK Crash Reporting and WebView Tracking usage it is needed also to add RUM and/or Logs artifacts to be able to report events to RUM and/or Logs respectively.
```

Comment:
```suggestion
**NOTE**: In case of NDK Crash Reporting and WebView Tracking usage it's also needed to add RUM and/or Logs artifacts to be able to report events to RUM and/or Logs respectively.
```